transforms/code/license_check/ray/pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[project]
+name = "dpk_license_check_transform_ray"
+version = "0.2.1.dev0"
+requires-python = ">=3.10"
+description = "License Check Transform"
+license = {text = "Apache-2.0"}
+readme = {file = "README.md", content-type = "text/markdown"}
+authors = [
+    { name = "David Wood", email = "dawood@us.ibm.com" },
```

Comment:
I thinkk these shold be the same names as in python's pyproject.toml

Reply:
done